docs/InterfaceKit.md

```diff
@@ -1,6 +1,6 @@
 #Phidget InterfaceKit Boards
-This module will work with all Phidget InterfaceKit I/O boards. Some boards like the relay and signal switching boards may not have inputs, triggers, or data rates, so these fields will be empty when connected to such boards.
 
-[Basic InterfacKit example](https://github.com/RIAEvangelist/node-phidget-API/blob/master/examples/InterfaceKit.js)
+The Phidget Manager library makes for an intuitive and easy way to manage your Phigets I/O devices. For a quick start into your Manager project see the [Getting Started](https://github.com/RIAEvangelist/node-phidget-API/blob/master/docs/InterfaceKit.md#GettingStarted) section or jump to this [basic I/O example](https://github.com/RIAEvangelist/node-phidget-API/blob/master/examples/InterfaceKit.js).
+
+This module will work with all Phidget InterfaceKit I/O boards. Some boards like the relay and signal switching boards may not have inputs, triggers, or data rates, so these fields will be empty when connected to such boards.
 
-ToDo : map out module and describe use
```

Comment:
had you left this line it would be good for commit signalling there was more work to be done.